activiti-cloud-runtime-bundle-service/activiti-cloud-services-runtime-bundle/activiti-cloud-services-rest-api/src/main/java/org/activiti/cloud/services/rest/api/ProcessInstanceVariableAdminController.java

```diff
@@ -25,11 +28,15 @@
 import org.springframework.web.bind.annotation.PutMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
 
 @RequestMapping(value = "/admin/v1/process-instances/{processInstanceId}/variables",
         produces = {MediaTypes.HAL_JSON_VALUE, MediaType.APPLICATION_JSON_VALUE})
 public interface ProcessInstanceVariableAdminController {
 
+    @RequestMapping(method = RequestMethod.GET)
```

Comment:
It looks this fix is also adding new functionality, can we reference some issue or add an explanation in the PR? Thanks

Reply:
it just exposes an existing endpoint that wasn't exposed before.